.github/workflows/create_release_pr.yml

```diff
@@ -20,6 +20,9 @@ jobs:
       - name: Sanitise workflow inputs # Should be 1st step
         uses: cylc/release-actions/stage-1/sanitize-inputs@v1
 
+      - name: Check CONTRIBUTING.md
+        uses: cylc/release-actions/check-shortlog@v1
```

Comment:
Here and in the shortlog.yml, I think I'll have to use either a new tag or a commit after this PR: https://github.com/cylc/release-actions/pull/18 @MetRonnie

Reply:
cylc/release-actions#18 is more of a bug fix imo, can keep the v1 tag*

---

*Actually it's a branch rather than tag - makes it easier to update. The idea is the branch always points to the latest commit that is still backwards compatible with the original state of the actions repo when v1 was ready to use. (We could also use tags for `major.minor.patch` releases, but it's not really needed.)

Reply:
Got it, let me rebase it now.